src/styles/_catalog.scss

```diff
@@ -1,22 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(auto-fill, 200px);
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 487px) {
+    display: grid;
```

Comment:
You don't need to set display: grid; in the media, it is already set to the main selector